### PR TITLE
Option to enable / disable stats that require external processes

### LINKED ISSUE #6613: [Support]: nethogs CPU usage
### Describe the problem you are having

I'm seeing nethogs use 20-40% CPU every few seconds (according to htop).

Would be nice to have an option to disable it for people who don't care too much about monitoring bandwidth.

### Version

0.13.0-53d63e0

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
n/a
```

### FFprobe output from your camera

```shell
n/a
```

### Frigate stats

_No response_

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

n/a

### Any other information that may be helpful

_No response_

### PR DESCRIPTION
**Pull request by NickM-27**
Closes #6613

### CODE REVIEW

**Comment by skrashevich** [comment]
it's overkill for resolve #6613 :)

**Comment by NickM-27** [comment]
Maybe, but I agree that many users (like myself) won't have use for these and there's no sense taking up extra time for stats which aren't needed, especially as more and more types of stats are added.

**Comment by blakeblackshear** [comment]
Perhaps we should expand this scope to allow disabling all stats that require spawning external processes. The system page used to be very fast. For example, gpu usage never really changes once things are up and running.

**Comment by skrashevich** [comment]
Absolutely agree. I was precisely endeavoring to minimize the invocation of external processes during the statistics calculations (#6298, #6344, #6359).

File: docs/docs/configuration/index.md [hunk 1]
```diff
@@ -558,6 +558,14 @@ ui:
 
 # Optional: Telemetry configuration
 telemetry:
+  # Optional: Configure system stats
+  stats:
+    # Enable AMD GPU stats (default: shown below)
+    amd_gpu_stats: true
```
Comment (blakeblackshear) [inline docs/docs/configuration/index.md:564]:
```suggestion
    amd_gpu_stats: True
```

File: docs/docs/configuration/index.md [hunk 2]
```diff
@@ -558,6 +558,14 @@ ui:
...
+    # Enable Intel GPU stats (default: shown below)
+    intel_gpu_stats: true
```
Comment (blakeblackshear) [inline docs/docs/configuration/index.md:566]:
```suggestion
    intel_gpu_stats: True
```

**Review by blakeblackshear — CHANGES_REQUESTED** [review]

**Review by blakeblackshear — APPROVED** [review]

**Pull request merged**